components/MnemonicView.tsx

```diff
@@ -65,6 +69,7 @@ const MnemonicInput: FC<MnemonicInputProps> = ({
             {orderNo}
           </Kbd>
           <Input
+            key={`mnemonic-phrase-view-input-item-${index}`}
```

Comment:
Why we need key here?

Reply:
For some reason, possibly because of top level key, got warnings from those inputs in storybook